### ISSUE TITLE
Translation bugs after vs update

### ISSUE THREAD

**Issue by schneiderbruno** [Microsoft/vscode#70245]
After new vs updates, translation buttons ares bugs... 
Multiples "&" have been addded.
Example: "Add pasta para área de trabalho", are "A & & dd área de trabalho".

Please fix it.

**Comment by dbaeumer** [comment]
A screen shot showing this:

<img width="520" alt="capture" src="https://user-images.githubusercontent.com/1931590/54267242-312e7b80-4579-11e9-85be-ee11147ba7da.png">

**Comment by dbaeumer** [comment]
@danyeh this seems to be a problem with the translation: The strings in the bundle look like this

```
"New &&Window","&&Arquivo","&&Editar","& & seleção","&&Exibir"
```

Observe `& & seleção`. Everything works fine for `New &&Window`. Can you please investigate why this is happening.

**Comment by dbaeumer** [comment]
This seems to be already broken here: https://github.com/Microsoft/vscode-loc/blob/master/i18n/vscode-language-pack-pt-BR/translations/main.i18n.json#L1

**Comment by dbaeumer** [comment]
@danyeh can you please check how that looks like in the xliff files. Or provide me with the latest export.

**Comment by janbaltus** [comment]
@shawndon In this case we need to communicate with community and explain the meaining of &amp;&amp; in MLPC. 

The space added between ampersands and even is added in MT suggestion should be fixed and not kept as it is.
![obrazek](https://user-images.githubusercontent.com/18324209/54424044-57d3ea00-4712-11e9-94b4-c3edca5918a6.png)
and if community get enough likes for that strings is is used.

**Comment by dbaeumer** [comment]
@janbaltus thanks for getting back. My recommendation then is to fix this at least temporarily in vscode-loc before this can be addressed in MLPC. Steps

- in vscode-loc search for `& &`.
- check the text and decide if it should be replaced with the correct `&&`
- republish the affected language packs

I did a quick search in vscode-loc and there seems to be 126 occurences in total. Affected langauges are: de, es, pt-br, fr

**Comment by janbaltus** [comment]
There is also issue with space after &&. We are fixing this as well

**Comment by schneiderbruno** [comment]
thx for all, for the atention in this question.

**Comment by schneiderbruno** [comment]
remember: this problem not ocurred in vs update ~1.2 ....

**Comment by dbaeumer** [comment]
@janbaltus did you publish new LPs?

**Comment by janbaltus** [comment]
accordign to schedule releas of LP is 3/20.
I can publish today instedad of Wednesday

**Comment by shawndon** [comment]
@janbaltus  I believe Dirk is referring to the March release 1.32

**Comment by dbaeumer** [comment]
@janbaltus yes, I mean updated language packs for 1.32.

**Comment by janbaltus** [comment]
MLCP tool is set to master branch, which now has strings for 1.33 version.
We are able to fix only this version strings in MLCP.

I just published 1.33 LP version.

**Comment by dbaeumer** [comment]
@janbaltus could you not simply branch the `vscode-loc` repository fix the json files and republish the LPs. This IMO doesn't need a new import from MLCP.

**Comment by janbaltus** [comment]
Published 1.32.5 LP version, but in case string with HK issue had also translation issue it show up now in English. There is no translation available yet for those strings.
Next final LP release will fix also untranslated issue.

**Comment by dbaeumer** [comment]
@janbaltus thanks. I will close this issue then.
